docs/conf.py

```diff
@@ -62,9 +62,9 @@
 # built documents.
 #
 # The short X.Y version.
-version = '1.0'
+version = '1.1'
 # The full version, including alpha/beta/rc tags.
-release = '1.0.0'
+release = '1.1.0'
```

Comment:
Hi! 

I use in my project the 'VERSION' file and when I want make a new release,
I just increment version number in 'VERSION' file and run [make newrelease](https://github.com/AndrewBurdyug/makechat/blob/master/Makefile#L127-L151), that's all! 

All rest(coverage, run tests, update docs) doing related services(TravisCI, CircleCI, ReadTheDocs, Codecov) which have been connected to github. 

Maybe this approach help you implement your own release management scenario ;)